opentelemetry-api/tests/metrics/test_metrics.py

```diff
@@ -35,6 +35,16 @@ def test_counter_add(self):
         counter = metrics.Counter()
         counter.add(1, {})
 
+    def test_updowncounter(self):
+        counter = metrics.UpDownCounter()
+        bound_counter = counter.bind({})
+        self.assertIsInstance(bound_counter, metrics.BoundCounter)
+
+    def test_updowncounter_add(self):
+        counter = metrics.Counter()
+        counter.add(1, {})
+        counter.add(-1, {})
```

Comment:
How is this test testing anything? :thinking:

Reply:
I more or less just copied from L36. Since the API versions should just be no-op (minimal implementation) in case no SDK is installed, I think the tests in this file just assert it accepts the calls as expected but not that it does anything?